### PR TITLE
Nukes Duplicate Code

### PR DESCRIPTION
**Pull request by jleandroperez**
/cc @diegoreymendez (Thanks in advance!)

### CODE REVIEW

File: WordPress/Classes/System/WordPressAppDelegate.m [hunk 1]
```diff
@@ -122,6 +121,9 @@ - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:
     // Start Simperium
     [self loginSimperium];
 
+    // Listen to Local Notifications
+    [self hookLocalNotifications];
```
Comment (diegoreymendez) [inline WordPress/Classes/System/WordPressAppDelegate.m]: I'd change the word "hook" with "observe" or "listen" which are the terms we commonly use.  I'd also consider removing the comment above as the call should make it's own purpose clear at that point.

File: WordPress/Classes/System/WordPressAppDelegate.m [hunk 2]
```diff
@@ -1238,9 +1202,16 @@ - (void)toggleExtraDebuggingIfNeeded
...
 
-#pragma mark - Notifications
+#pragma mark - Local Notifications Helpers
 
-- (void)handleDefaultAccountChangedNotification:(NSNotification *)notification
+- (void)hookLocalNotifications
+{
+    NSNotificationCenter *nc = [NSNotificationCenter defaultCenter];
+    [nc addObserver:self selector:@selector(handleDefaultAccountChangedNote:) name:WPAccountDefaultWordPressComAccountChangedNotification object:nil];
+    [nc addObserver:self selector:@selector(handleLowMemoryWarningNote:) name:UIApplicationDidReceiveMemoryWarningNotification object:nil];
```
Comment (diegoreymendez) [inline WordPress/Classes/System/WordPressAppDelegate.m]: Breaking this line would make it fit the 80 chars limit.

**Comment by diegoreymendez** [comment]
Travis report the build is failing but I'm able to build this locally.

**Comment by diegoreymendez** [comment]
:shipit:

**Comment by jleandroperez** [comment]
@diegoreymendez updated!. For consistency's sake, let's leave the `// Local Notifications` comment. (Almost) everything that's initialized there is grouped by section.

Re: Travis, i've been seeing it failing in several PR's, when the build is actually working. I think we should reconsider that integration, since it's bringing more issues than benefits.

Thanks!

**Pull request merged**